client/jetpack-cloud/sections/agency-dashboard/style.scss

```diff
@@ -1,126 +1,3 @@
-/**
- * The currently selected site block
- * at the top of the sidebar in My Sites
- *
- */
-
-.current-site.card {
-	margin: 0;
-	padding: 0;
-	box-shadow: none;
-	background: transparent;
-
-	&.is-loading,
-	&.is-no-sites {
-		.site__title {
-			color: var(--color-neutral-50);
-			line-height: 35px;
-		}
-
-		.current-site__switch-sites {
-			cursor: default;
-
-			&::before {
-				visibility: hidden;
-			}
-		}
-	}
-
-	&.is-loading .site-icon {
-		animation: pulse-light 0.8s ease-in-out infinite;
-	}
-}
-
-.current-site .site {
-	transition: opacity 0.15s ease-in-out;
-
-	.site__info {
-		animation: appear 0.3s ease-in-out;
-	}
-
-	.focus-sites & {
-		opacity: 0.2;
-		pointer-events: none;
-	}
-}
-
-.current-site {
-	.site,
-	.all-sites {
-		background: var(--color-sidebar-background);
-		border-bottom: 1px solid var(--color-sidebar-border);
-	}
-}
-
-.current-site__switch-sites {
-	display: block;
-	background: var(--color-sidebar-background);
-	border-bottom: 1px solid var(--color-sidebar-border);
-
-	.button.is-borderless {
-		display: block;
-		width: 100%;
-		text-align: left;
-		padding: 12px 8px 12px 44px;
-		position: relative;
-		font-weight: normal;
-		color: var(--color-sidebar-text);
-
-		.gridicon {
-			height: 24px;
-			width: 24px;
-			fill: var(--color-sidebar-gridicon-fill);
-			position: absolute;
-			top: 12px;
-			left: 14px;
-		}
-	}
-
-	&:hover {
-		background-color: var(--color-sidebar-menu-hover-background);
-
-		.button.is-borderless:hover,
-		.button.is-borderless:focus {
-			background-color: var(--color-sidebar-menu-hover-background);
-			color: var(--color-sidebar-menu-hover-text);
-
-			.gridicon {
-				fill: var(--color-sidebar-menu-hover-text);
-			}
-		}
-	}
-}
-
-.sidebar .notice.is-compact {
```

Comment:
In my tests, there's no `.sidebar` in either a4a or jetpack cloud where this style seem to be loaded, so I don't think this has any impact.